extensions/database/src/main/java/com/google/android/agera/database/SqlRequestCompiler.java

```diff
@@ -112,16 +121,40 @@ public Object arguments(@NonNull final String... arguments) {
 
```

Comment:
Also, `onConflict(int)` would only confuse, since it allows for more values than the request could really take. the compiler encapsulates (somewhat) this confusion